config/initializers/merit.rb

```diff
@@ -1,34 +1,9 @@
-# Use this hook to configure merit parameters
 Merit.setup do |config|
-  # Check rules on each request or in background
-  # config.checks_on_each_request = true
 
-  # Define ORM. Could be :active_record (default) and :mongoid
-  # config.orm = :active_record
-
-  # Add application observers to get notifications when reputation changes.
-  # config.add_observer 'MyObserverClassName'
-
-  # Define :user_model_name. This model will be used to grand badge if no
-  # `:to` option is given. Default is 'User'.
-  # config.user_model_name = 'User'
-
-  # Define :current_user_method. Similar to previous option. It will be used
-  # to retrieve :user_model_name object if no `:to` option is given. Default
-  # is "current_#{user_model_name.downcase}".
-  # config.current_user_method = 'current_user'
 end
 
-# Create application badges (uses https://github.com/norman/ambry)
+badges = [{ id: 1, name: 'checked_in', custom_fields: {image: "/shooting-star.png"} }, {id: 2, name: 'checked_inx2', custom_fields: {image: "/ruby heart.png"} }]
```

Comment:
Line is too long. [161/80]<br>Space inside { missing.<br>Space inside } missing.